tests/e2e/components_test.go

```diff
@@ -258,6 +258,14 @@ func (c *ComponentTestCtx) ValidateCRDReinstated(t *testing.T, name string) {
 	g.List(gvk.CustomResourceDefinition, crdSel).Eventually().Should(
 		HaveLen(1),
 	)
+	if len(version) != 0 {
+		g.Get(
+			gvk.CustomResourceDefinition,
+			types.NamespacedName{Name: name},
+		).Eventually(5*time.Second, 500*time.Millisecond).Should(
+			jq.Match(`.status.storedVersions[0] == "%s"`, version[0]),
```

Comment:
I think we should probably check all the version passed a an arg, right ?

Reply:
```go
slices.Sort(versions)
v := strings.Join(",", versions)

jq.Match(`.status.storedVersions | sort | join(",") == "%s"`, v)
```

Reply:
if  CRD has 
```
.status.storedVersions:
- v1alpha1
- v1beta1
```

then i will need to pass both 2 versions(orderless), not just `v1beta1` right?

in multikueue case, it only accept one version, so, this should not happen i guess.

Reply:
> then i will need to pass both 2 versions(orderless), not just v1beta1 right?

correct

> in multikueue case, it only accept one version, so, this should not happen i guess.

correct, but this is a shared test, so the expectation is tho check all the versions passed, I guess. if you only want to check one specific version, then the test should probably be done in a different way

Reply:
you are right, to get it a generic test, need to cover this case.